utils/ansible-freeipa.spec.in

```diff
@@ -0,0 +1,75 @@
+# Turn off automatic python byte compilation because these are Ansible
+# roles and the files are transferred to the node and compiled there with
+# the python verison used in the node
+%define __brp_python_bytecompile %{nil}
+
+Summary: Roles and playbooks to deploy FreeIPA servers, replicas and clients
+Name: ansible-freeipa
+Version: @@VERSION@@
+Release: @@RELEASE@@%{?dist}
+URL: https://github.com/freeipa/ansible-freeipa
+License: GPLv3+
+Source: %{name}-%{version}-@@RELEASE@@.tar.bz2
+BuildArch: noarch
+
+%description
+ansible-freeipa provides Ansible roles and playbooks to install and uninstall
+FreeIPA servers, replicas and clients. Also modules for group, host, topology
+and user management.
+
+Note: The ansible playbooks and roles require a configured ansible environment
+where the ansible nodes are reachable and are properly set up to have an IP
+address and a working package manager.
+
+%package tests
+Summary: ansible-freeipa tests
+Requires: %{name}
+
+%description tests
+
+%prep
+%setup -q -n ansible-freeipa-%{version}-@@RELEASE@@
+# Do not create backup files with patches
+# Fix python modules and module utils:
+# - Remove shebang
+# - Remove execute flag
+for i in roles/ipa*/library/*.py roles/ipa*/module_utils/*.py plugins/*/*.py; do
+    sed -i '/\/usr\/bin\/python*/d' $i
```

Comment:
What if we have `/usr/bin/python` in other places on the files?
Wouldn't it be better to use `sed -i '1{/\/usr\/bin\/python*/d;}' $i`?